rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -93,13 +93,11 @@ void main_window::Init()
 	Q_EMIT RequestGlobalStylesheetChange(m_gui_settings->GetCurrentStylesheetPath());
 	ConfigureGuiFromSettings(true);
 
-#ifdef BRANCH
-	if ("RPCS3/rpcs3/master"s != STRINGIZE(BRANCH) && ""s != STRINGIZE(BRANCH))
-#else
-	if (false)
-#endif
+	std::string_view branch_name = rpcs3::get_full_branch();
+
+	if (bool is_master = branch_name == "RPCS3/rpcs3/master"s; !is_master)
```

Comment:
```suggestion
	if (branch_name != "RPCS3/rpcs3/master"s)
```

Reply:
I would have done that to begin with, but `!=` is actually deprecated already. I might just put it in parens tho.

Reply:
Ahh no, I was mistaken. Sorry for that.